pydm/utilities/colors.py

```diff
@@ -0,0 +1,25 @@
+import pickle
+import os
+import itertools
+
+current_dir = os.path.dirname(os.path.realpath(__file__))
+
+svg_color_to_hex_map = None
+hex_to_svg_color_map = None
+with open(os.path.join(current_dir, 'hex2color.pkl')) as f:
+  hex_to_svg_color_map = pickle.load(f)
```

Comment:
This line lead to error:  `TypeError: a bytes-like object is required, not 'str'`

Opening the file in binary mode solved it. I mean:
`with open(os.path.join(current_dir, 'hex2color.pkl'), 'rb') as f:`